Day: 20130624
41294 logs on server
Downloading 41294 logs from server
./dbupdate.sh: line 45: /home/ai/dominion/code/db/log2db.py: No such file or directory
Day: 20130625
11193 logs on server
Downloading 11193 logs from server
20130625
Traceback (most recent call last):
  File "./log2db.py", line 21, in <module>
    day_str = m.group(1)
AttributeError: 'NoneType' object has no attribute 'group'
